config/crd/kustomization.yaml

```diff
@@ -5,5 +5,6 @@ resources:
 - bases/numaplane.numaproj.io_pipelinerollouts.yaml
 - bases/numaplane.numaproj.io_numaflowcontrollerrollouts.yaml
 - bases/numaplane.numaproj.io_isbservicerollouts.yaml
+- bases/numaplane.numaproj.io_monovertexrollouts.yaml
 # install numaflow minimal CRDs as a dependency
 - https://github.com/numaproj/numaflow/config/advanced-install/minimal-crds?ref=stable
```

Comment:
Add Numaflow MonoVertex CRD here

Reply:
The minimal CRD for MonoVertex is already included here: https://github.com/numaproj/numaflow/tree/main/config/base/crds/minimal

Perhaps all we need to do is remove Numaflow CRDs from the Makefile then.